src/classes/SiblingPages.php

```diff
@@ -20,6 +20,7 @@ public function __construct() {
   public function widget($args, $instance) {
     echo $args['before_widget'];
 
+    //checks show_parent
```

Comment:
👍 Good call, this should have a comment.  I'd suggest being a little more descriptive, this could work:
```
// Checks to see if the option to include the parent page in the list of sibling
// pages has been set
```